pkg/metrics/client_go_adapter.go

```diff
@@ -119,34 +119,53 @@ var (
 	}, []string{"name"})
 )
 
-func init() {
-	registerClientMetrics()
-	registerReflectorMetrics()
-}
+// RegisterClientMetrics sets up the client latency metrics from client-go.
+// To opt *out* of specific metrics, specify their *Key consts to optOut.
```

Comment:
Realistically this is lousy UX without a better indication that the varargs are opt *out* rather than *in*. It would probably be better to flip it.